panel/src/components/Forms/Input/SlugInput.vue

```diff
@@ -54,7 +47,6 @@ export const props = {
  * @example <k-input :value="slug" @input="slug = $event" name="slug" type="slug" />
  */
 export default {
-	extends: TextInput,
```

Comment:
By no longer extending the TextInput, we lose the focus method.

Reply:
I think by extending now `StringInput` (which I added again), this should be fixed.

Reply:
Yep, works great now.